### DISCUSSION TITLE
New Strategy

### DISCUSSION THREAD

**Discussion by Jaszczur11** [lmusarella/Pancakeswap-Prediction-Bot-2023#21] [Ideas]
Hello, I have a question if a colleague is still working on the script something? I found a script with a thief code, but the strategy looks different. I played a bit and it seems to me that the strategy there is better at predicting UP or Down. Is it possible to adapt this strategy to your script? I personally tried, but without success.
Link: https://github.com/PancakeSwapPredictionBot/PancakeswapPredictionBot-2023
Greetings

**Comment by lmusarella** [comment]
Hi, the code you shared with me is what I used to create my bot. My bot has resumed in the same way the only strategy of the starting bot (signal strategy). Don't use the bot from that repository that still has the malicious code! If you like put a star to my repository :D
